frontend/lib/src/components/shared/Toolbar/styled-components.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import styled, { StyledComponent } from "@emotion/styled"
+
+import { hasLightBackgroundColor } from "@streamlit/lib/src/theme"
+
+export interface StyledToolbarWrapperProps {
+  locked?: boolean
+  target?: StyledComponent<any, any, any>
+}
+
+export const StyledToolbarWrapper = styled.div<StyledToolbarWrapperProps>(
+  ({ locked, target }) => ({
+    padding: "0.5rem 0 0.5rem 0.5rem",
+    position: "absolute",
+    top: locked ? "-2.4rem" : "-1rem",
+    right: "0rem",
```

Comment:
super nit: should this be taking in the theme and then doing `theme.spacing.none`?

Reply:
Updated